### PR TITLE
fix lint warnings in newsreaderExample

### PR DESCRIPTION
**Pull request by zaki50**
@realm/java

### CODE REVIEW

File: examples/encryptionExample/lint.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<lint>
+    <issue id="AllowBackup" severity="ignore" />
+    <issue id="IconLauncherShape" severity="ignore" />
+    <issue id="IconMissingDensityFolder" severity="ignore" />
+    <issue id="GoogleAppIndexingWarning" severity="ignore" />
+    <issue id="PrngFix" severity="ignore" />
```
Comment (nhachicha) [inline examples/encryptionExample/lint.xml]: remove please

File: examples/newsreaderExample/lint.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<lint>
+    <issue id="AllowBackup" severity="ignore" />
+    <issue id="IconLauncherShape" severity="ignore" />
+    <issue id="IconMissingDensityFolder" severity="ignore" />
+    <issue id="GoogleAppIndexingWarning" severity="ignore" />
+    <issue id="PrngFix" severity="ignore" />
```
Comment (nhachicha) [inline examples/newsreaderExample/lint.xml]: remove please

File: examples/encryptionExample/lint.xml (hunk 1 above)
Comment (zaki50) [inline examples/encryptionExample/lint.xml]: done

**Comment by cmelchior** [comment]
:+1:

**Comment by stk1m1** [comment]
👍

**Pull request merged**